hail/python/hailtop/batch/docs/docker_resources.rst

```diff
@@ -71,29 +71,26 @@ For more information about Dockerfiles and directives that can be used see the f
 Building Images
 ---------------
 
-To create a Docker image, we use a series of commands to build the image from a Dockerfile by specifying
-the context directory (in this case the current directory `.`). The `-f` option
-specifies what Dockerfile to read from. The `-t` option is the name of the image.
-More in depth information can be found `here <https://docs.docker.com/engine/reference/commandline/build/>`__.
+To create a Docker image, use `docker build <dir> -t <name> -f <dockerfile>`. <dir> specifies the context directory (typically `.`, for the current working directory).
+Use `-t` to name the image, and `-f` to specify the Dockerfile path.
+More information can be found `here: <https://docs.docker.com/engine/reference/commandline/build/>`__.
 
 .. code-block:: sh
 
-    docker build -t <my-image> -f Dockerfile .
+    docker build -t gcr.io/<my-project>/<my-image>:<tag> -f Dockerfile .
 
 
+In this example we prepend the image name with `gcr.io/<my-project>/` so that it may be pushed to the Google Container Registry, in the next step.
+
 Pushing Images
 --------------
 
-To use an image with Batch, you need to upload your image to a place where Batch can access it.
-You can store images inside the `Google Container Registry <https://cloud.google.com/container-registry/docs/>`__ in
-addition to Dockerhub. Below is an example of pushing the image to the Google Container Registry.
-It's good practice to specify a tag that is unique for your image. If you don't tag your image, the default is
-`latest`. For more information about pulling and pushing images for the Google Container Registry, see
+To use an image with Batch, you need to upload it somewhere Batch can read it: either the `Google Container Registry <https://cloud.google.com/container-registry/docs/>`__ or
+Dockerhub. Below is an example of pushing the image to the Google Container Registry. For more information about pulling and pushing images for the Google Container Registry, see
```

Comment:
I think the docker folks prefer the styling "Docker Hub," do you mind fixing that while we're editing this file anyway? Thanks!

Reply:
Confusing:

<img width="163" alt="Screen Shot 2020-08-06 at 5 47 07 AM" src="https://user-images.githubusercontent.com/5543229/89517808-45939980-d7a8-11ea-9eab-a6c103fc8e22.png">